userguide/account.rst

```diff
@@ -83,31 +83,31 @@ summarizes the available options when creating a group.
    |                     |           |                                                                                                                          |
    |                     |           |                                                                                                                          |
    +=====================+===========+==========================================================================================================================+
-   | Group ID            | string    | the next available group ID will be suggested for you; by convention, UNIX groups containing user accounts have          |
-   |                     |           | an ID greater than 1000 and groups required by a service have an ID equal to the default port number used by the         |
-   |                     |           | service (e.g. the sshd group has an ID of 22)                                                                            |
+   | GID                 | string    | The next available group ID is suggested. By convention, UNIX groups containing user accounts have an ID greater than    |
```

Comment:
Why the added bold on these numbers?

Reply:
I added bold to emphasise the values used by the convention, but I'll remove it.